src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/Data/CollectionViewGroupInternal.cs

```diff
@@ -867,7 +852,7 @@ public void RemoveEmptyGroup(CollectionViewGroupInternal group)
             {
                 if (_toRemove == null)
                 {
-                    _toRemove = new System.Collections.Generic.List<CollectionViewGroupInternal>();
+                    _toRemove = new List<CollectionViewGroupInternal>();
```

Comment:
You can remove the namespace for the definition of `_toRemove` as well.

Reply:
Nice catch, will do.